v0/destinations/hs/HSTransform-v1.js

```diff
@@ -202,11 +283,52 @@ const legacyBatchEvents = destEvents => {
           batchedResponse.destination
         )
       );
+    } else if (event.message.source && event.message.source === "rETL") {
+      const { endpoint } = event.message;
+      maxBatchSize = endpoint.includes("contact")
+        ? MAX_BATCH_SIZE_CRM_CONTACT
+        : MAX_BATCH_SIZE_CRM_OBJECT;
+      const { operation } = event.message;
+      if (operation) {
+        if (operation === "createObject") {
+          createAllObjectsEventChunk.push(event);
+        } else if (operation === "updateObject") {
+          updateAllObjectsEventChunk.push(event);
+        }
+      } else {
+        throw new CustomError("[HS]:: Error in getting operation", 400);
```

Comment:
nit: Should we add `rETL` in all error message which can generate from rETL source? This can help in many ways to classify and pinpoint the errors easily.
```suggestion
        throw new CustomError("[HS]:: rETL - Error in getting operation", 400);
```